src/openforms/js/components/admin/form_design/variables/ServiceFetchConfigurationPicker.stories.mdx

```diff
@@ -0,0 +1,83 @@
+import {ArgsTable, Canvas, Meta, Story} from '@storybook/addon-docs';
+
+import {FormLogicContext} from '../Context';
+import ServiceFetchConfigurationPicker from './ServiceFetchConfigurationPicker';
+
+<Meta
+  title="Form design/ServiceFetchConfigurationPicker"
+  component={ServiceFetchConfigurationPicker}
+/>
+
+# ServiceFetchConfigurationPicker
+
+Form that is shown when adding a ServiceFetchConfiguration. The user can opt to select an existing
+configuration and copy/update it; or to create a new configuration from scratch.
+
+export const Template = args => (
+  <FormLogicContext.Provider
+    value={{
+      services: [
+        {
+          label: 'Service 1',
+          apiRoot: 'http://foo.com/api/v1/',
+          apiType: 'ORC',
+        },
+        {
+          label: 'Service 2',
+          apiRoot: 'http://bar.com/api/v1/',
+          apiType: 'ORC',
+        },
+      ],
+      serviceFetchConfigurations: [
+        {
+          name: 'Foo fetch',
+          url: 'http://openforms.dev/service-fetch-configurations/foo',
+          service: 'http://foo.com/api/v1/',
+          path: '/some-path',
+          method: 'GET',
+          headers: [['X-Foo', 'foo']],
+          queryParams: [['parameter2', ['value1', 'value2']]],
+          body: {field1: 'value', field2: 'value2'},
+          dataMappingType: 'jq',
+          mappingExpression: '.field.nested',
+        },
+        {
+          name: '', // No name supplied, should fall back to "method path (service)"
+          url: 'http://openforms.dev/service-fetch-configurations/bar',
+          service: 'http://bar.com/api/v1/',
+          path: '/some-other-path',
+          method: 'POST',
+          headers: [['X-Foo', 'bar']],
+          queryParams: [
+            ['parameter', ['value']],
+            ['parameter2', ['value1', 'value2']],
+          ],
+          body: {field1: 'value', field2: 'value2'},
+          dataMappingType: 'JsonLogic',
+          mappingExpression: {var: 'field'},
+        },
+      ],
+    }}
+  >
+    <ServiceFetchConfigurationPicker {...args} />
+  </FormLogicContext.Provider>
+);
+
+## Presentation
+
+<Canvas>
+  <Story
+    name="Default"
+    args={{
+      onChange: () => {
+        null;
+      },
```

Comment:
probably `jest.fn(...)` is better

Reply:
I changed it to actually accept the arguments that are passed, but I'm not sure what `jest.fn` would do in this case?